main.py

```diff
@@ -90,6 +92,26 @@ def birthdays(contacts: AddressBook):
     return contacts.get_birthdays_per_week()
 
 
+def add_address(args, contacts: AddressBook):
+    try:
+        name = args[0]
+        address = (" ").join(args[1:]).title()
+        if not address.strip():
+            raise ValueError
+        contact = contacts.find(name)
+        contact.add_address(address)
+        contacts.save_records()
+        return 'Address added.'
+    except ValueError:
+        return 'You need to give name and address (minimum 1 word).'
```

Comment:
Please check https://github.com/pavlomatchak/team-project-group-4/blob/a764a062ba09ed7209d1a5e7133e684343ed05bf/main.py#L31-L40 and https://github.com/pavlomatchak/team-project-group-4/blob/a764a062ba09ed7209d1a5e7133e684343ed05bf/phone.py#L3-L14
and try to do the same :) It's better to keep logic for specific fields separate from main code